src/lib/parse/CompoundValueNode.h

```diff
@@ -43,7 +43,7 @@ namespace orion
 * The class fields:
 * -------------------------------------------------------------------------------
 * o name         When parsing an XML payload, each node in the tree has a tag.
-*                The name of the node is taken from the tag-name.
+*                The name of the node is taken from the tag-name. (FIXME P4 #1862: reference to XML in this comment is obsolete)
```

Comment:
Suggestion:

    * o name         Note that not all nodes have a name, not in JSON payloads, nor when
    *                getting the info from mongo BSON.